publisher/publisher.go

```diff
@@ -233,9 +231,11 @@ func downloadFile(url, destPath string) error {
 func downloadArtifacts(conf config, schema uploadArtifactsSchema) error {
 	for _, artifactSchema := range schema {
 		for _, arch := range artifactSchema.Arch {
-			err := downloadArtifact(conf, artifactSchema.Src, arch)
-			if err != nil {
-				return err
+			for _, sysMngr := range artifactSchema.SysManager {
+				err := downloadArtifact(conf, artifactSchema.Src, arch, sysMngr)
```

Comment:
Would this work in case no sys_manager is specified? (I.e. for the integrations)

Reply:
Ok I saw that a different PR fixes this issue adding an empty sysManager.